File: backend/models/Jogo.cs
namespace backend.models
{
     public class Jogo
     {
          public int Id { get; set; }
          public int Placar { get; set; }
          public int PontuacaoMinimaTemporada { get; set; }
          public int PontuacaoMaximaTemporada { get; set; }
          public int QuantidadeRecordMinimoQuebrado { get; set; }
          public int QuantidadeRecordMaximoQuebrado { get; set; }

          public Jogo() { }

          public Jogo(int id, int placar, int pontuacaoMinimaTemporada, int pontuacaoMaximaTemporada, int quantidadeRecordMinimoQuebrado, int quantidadeRecordMaximoQuebrado)
          {
               Id = id;
               Placar = placar;
               PontuacaoMinimaTemporada = pontuacaoMinimaTemporada;
               PontuacaoMaximaTemporada = pontuacaoMaximaTemporada;
               QuantidadeRecordMinimoQuebrado = quantidadeRecordMinimoQuebrado;
               QuantidadeRecordMaximoQuebrado = quantidadeRecordMaximoQuebrado;
          }
     }
}

File: frontend/src/app/models/Jogo.ts
export class Jogo {
  id?: number;
  placar?: number;
  pontuacaoMinimaTemporada: number;
  pontuacaoMaximaTemporada: number;
  quantidadeRecordMinimoQuebrado: number;
  quantidadeRecordMaximoQuebrado: number;

  constructor() {
    this.id = 0;
    this.placar = 0;
    this.pontuacaoMinimaTemporada = 0;
    this.pontuacaoMaximaTemporada = 0;
    this.quantidadeRecordMinimoQuebrado = 0;
    this.quantidadeRecordMaximoQuebrado = 0;
  }
}
